dojo/endpoint/signals.py

```diff
@@ -0,0 +1,30 @@
+from auditlog.models import LogEntry
+from django.conf import settings
+from django.contrib.contenttypes.models import ContentType
+from django.dispatch import receiver
+from django.db.models.signals import post_delete
+from django.utils.translation import gettext as _
+from django.urls import reverse
+
+from dojo.models import Endpoint
+from dojo.notifications.helper import create_notification
+
+
+@receiver(post_delete, sender=Endpoint)
+def endpoint_post_delete(sender, instance, using, origin, **kwargs):
+    if instance == origin:
+        if settings.ENABLE_AUDITLOG:
+            le = LogEntry.objects.get(
+                    action=LogEntry.Action.DELETE,
+                    content_type=ContentType.objects.get(app_label='dojo', model='endpoint'),
+                    object_id=instance.id
+            )
+            description = _('The endpoint "%(name)s" was deleted by %(user)s') % {
+                                'name': str(instance), 'user': le.actor}
```

Comment:
I'm not sure how `le.actor` is populated here, but it says the endpoint was deleted by "None" whether I delete it via the API or the UI when acting as the `admin` user. I created a new user `test` and tried it again. This time I still get `None` when deleting via the API, but no notification at all when deleting from the UI. I'm not sure what's happening there...

![Screenshot 2024-04-09 at 20 21 40](https://github.com/DefectDojo/django-DefectDojo/assets/1749665/9e13d710-bc8c-48e8-be68-cbdf9d5edda5)